loader/loader.go

```diff
@@ -1272,8 +1420,18 @@ func (l *Loader) restoreData(ctx context.Context) error {
 				dispatchMap[fmt.Sprintf("%s_%s_%s", db, table, file)] = j
 			}
 		}
+
+		for view := range views {
```

Comment:
Is it possible to create the view in a different database with the table?

Reply:
I guess user may create that type of view. And dumpling will generate a "table file" for each view at view's database, I guess this logic is fine?

Reply:
done in https://github.com/pingcap/dm/pull/1310/commits/d16871614a6202d7f5414827a86276e1bf1dbcba